openmetadata-ui/src/main/resources/ui/src/styles/components/button.less

```diff
@@ -5,5 +5,6 @@ button {
     img {
       opacity: 0.5;
     }
+    height: 32px;
```

Comment:
This is not a good thing to add. It will restrict height to 32px for all the buttons with disabled prop. 
Move this to less impacted area